interp/constrextern.ml

```diff
@@ -325,16 +333,16 @@ let is_zero s =
   in aux 0
 
 let make_notation_gen loc ntn mknot mkprim destprim l bl =
-  match ntn,List.map destprim l with
+  match snd ntn,List.map destprim l with
```

Comment:
Not a fan of all the `snd/fst` added here; would it be too much of a pain to turn this into a record?

Reply:
If you insist very strongly, I can try, but to tell the truth I'm a bit exhausted by this PR and I would prefer that such a change is done at a later step when there are less other things to finish.

Reply:
Yes, retrieving information in the ocean of PR comments is difficult. Maybe adding the question of dependencies internal to `interp` in the [Maintenance](https://github.com/coq/coq/projects/8) or [API](https://github.com/coq/coq/projects/8) project, and the question of unifying `constr` and `custom` in a `Notations` project?

Reply:
Whereas I encourage you to use projects to organize (part of) your work, projects without a clear intent and without regular updates like the two you mentioned are pretty useless. So opening issues, possibly with "help wanted" labels seems like a better solution.